### DISCUSSION TITLE
How to disable scroll wheel zoom in ApexCharts?

### DISCUSSION THREAD

**Discussion by gespinha** [apexcharts/apexcharts.js#4732] [Q&A]
Hi everyone,

I've got this line chart in my app using ApexCharts:

```
import ApexCharts from "apexcharts";

var options = {
  chart: {
    type: "line",
  },
  series: [
    {
      name: "sales",
      data: [30, 40, 35, 50, 49, 60, 70, 91, 125],
    },
  ],
  xaxis: {
    categories: [1991, 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999],
  },
};

var chart = new ApexCharts(document.querySelector("#chart"), options);

chart.render();
```

[Example](https://codesandbox.io/p/sandbox/apexcharts-bug-twqqdn)

When I hover the chart and scroll using the mouse wheel, the chart zooms in/out. How do I disable this behavior while still allowing to select areas of the chart to Zoom (just disable the scroll wheel behavior)?

Thanks in advance!

**Comment by peshal-jha-work** [comment]
Try this

```
chart: {
        type: 'line',  
        zoom: {
          enabled: true,
          allowMouseWheelZoom: false
        }
      },
```

**allowMouseWheelZoom** is the key you want
